public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js

```diff
@@ -25,14 +25,14 @@ export class TimelineItem extends React.Component {
   render () {
     const value = this.props.fieldValue || {
       title: "",
-      date: Date.now(),
+      date: "",
       body: ""
     };
     return (
       <div className="form__field form__field--nested">
         <ManagedForm data={value} updateData={this.updateItem} onFormErrorsUpdate={this.props.onFormErrorsUpdate} formName="timelineEditor">
-        <ManagedField fieldLocation="date" name="Date" isRequired={true}>
-          <FormFieldDateInput isOutsideRange={() => false}/>
+        <ManagedField fieldLocation="date" name="Date" isRequired={true} customValidation={[isValidTimelineDate]}>
+          <FormFieldTextInput fieldPlaceholder='dd/mm/yyyy'/>
```

Comment:
Does this end up in the database as milliseconds? I can't work out where the conversion is done